src/DataGridExtensionsSample/Views/Customized2View.xaml

```diff
@@ -11,162 +11,166 @@
-    </Style>
-    <Style x:Key="DataGridTextBlockStyle" TargetType="TextBlock">
-      <Setter Property="Foreground" Value="Blue" />
-    </Style>
-    <ControlTemplate x:Key="IntegerFilter">
-      <controls:IntegerGreatherThanFilterControl Filter="{Binding Path=Filter, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType=dgx:DataGridFilterColumnControl}}" />
-    </ControlTemplate>
-    <ControlTemplate x:Key="FilterWithPopup">
-      <controls:FilterWithPopupControl Filter="{Binding Path=Filter, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType=dgx:DataGridFilterColumnControl}}" />
-    </ControlTemplate>
-    <ControlTemplate x:Key="MultipleChoiceFilter">
-      <dgx:MultipleChoiceFilter Filter="{Binding Path=Filter, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType=dgx:DataGridFilterColumnControl}}" />
-    </ControlTemplate>
-    <ControlTemplate x:Key="ComboboxFilter">
-      <Grid d:DataContext="{d:DesignInstance dgx:DataGridFilterColumnControl}">
-        <Grid>
-          <Control Style="{DynamicResource {x:Static dgx:DataGridFilter.IconStyleKey}}" />
-          <ComboBox x:Name="ComboBox" Text="{Binding Path=Filter, UpdateSourceTrigger=PropertyChanged}" ItemsSource="{Binding SourceValues}" IsEditable="True" />
-        </Grid>
-      </Grid>
-      <ControlTemplate.Triggers>
-        <Trigger SourceName="ComboBox" Property="Text" Value="">
-          <Setter TargetName="ComboBox" Property="Opacity" Value="0" />
-        </Trigger>
-        <Trigger SourceName="ComboBox" Property="Text" Value="{x:Null}">
-          <Setter TargetName="ComboBox" Property="Opacity" Value="0" />
-        </Trigger>
-        <Trigger SourceName="ComboBox" Property="IsMouseOver" Value="True">
-          <Setter TargetName="ComboBox" Property="Opacity" Value="1" />
-        </Trigger>
-        <Trigger SourceName="ComboBox" Property="IsFocused" Value="True">
-          <Setter TargetName="ComboBox" Property="Opacity" Value="1" />
-        </Trigger>
-      </ControlTemplate.Triggers>
-    </ControlTemplate>
-    <XmlDataProvider x:Key="ComboBoxSource" Source="../SampleList.xml" XPath="Items/Item" />
-  </UserControl.Resources>
-  <DockPanel>
-    <DockPanel DockPanel.Dock="Top" Margin="10">
-      <Button x:Name="ClearIpsumButton" DockPanel.Dock="Right" VerticalAlignment="Top" Padding="5,2" Command="{Binding ClearIpsumCommand}">Clear Ipsum</Button>
-      <Button x:Name="OpenAndPopulateAFilterButton" DockPanel.Dock="Right" VerticalAlignment="Top" Padding="5,2" Margin="0,0,10,0" Command="{Binding OpenAndPopulateAFilterCommand}">Open &amp; Populate a Filter</Button>
-      <Button x:Name="ProgrammaticAccessToFilterControlButton" DockPanel.Dock="Right" VerticalAlignment="Top" Padding="5,2" Margin="0,0,10,0" Command="{Binding ProgrammaticAccessToFilterControlCommand}">Change Popup Caption</Button>
-      <TextBlock TextWrapping="WrapWithOverflow">
+    <UserControl.Resources>

```

Comment:
90% of this PR is just reformatting the code. Please stick with the existing formatting.